### ISSUE TITLE
Add student mapping module

### ISSUE THREAD

**Issue by iamareebjamal** [zhcet-amu/zhcet-web#259]
Student Floated Course Registration module to be implemented in Admin Panel

**Comment by jagadeeshkoka** [comment]
I would like to work on this enhancement, can You tell where  I need to start ? I am not able to do pull request.

**Comment by iamareebjamal** [comment]
I'm sorry but are you aware of the working and scope of this project?

**Comment by jagadeeshkoka** [comment]
I am not sure what is the scope of this project, I tried to something of this kind recently, so I am familiar with the project. Do you have any requirement specification for this ??

**Comment by iamareebjamal** [comment]
Yes, we need to upload a CSV with this structure -

course_code, faculty_no 


And it should be parsed and validated. And corresponding students should be enrolled in the supplied courses

**Comment by jagadeeshkoka** [comment]
Oh which means, you are trying to register students in bulk mode? And only two fields in that CSV file? how about rest of the fields like Student nam, Student ID, ..etc

**Comment by jagadeeshkoka** [comment]
And I have checked out the project, there are a lot of compilation errors,  is this code working code ? or do you have any doc for running this project?

**Comment by iamareebjamal** [comment]
There's no compilation error as the builds are automatically generated, tested and deployed using CI

What compilation error did you face?

**Comment by jagadeeshkoka** [comment]
Some of the classes extended BaseEntity class which is abstract and subclasses didn't implement those methods.

**Comment by iamareebjamal** [comment]
Apply lombok plugin to your IDE

**Comment by jagadeeshkoka** [comment]
I am using eclipse do you recommend, I cant find that plugin though . do you recommend anyother ?

**Comment by jagadeeshkoka** [comment]
> I am using eclipse do you recommend, I cant find that plugin though . do you recommend anyother ?

Yeah its working i have added Lombok plugin

**Comment by jagadeeshkoka** [comment]
application UP and running, what is the login or any sample data SQL dump file to login and check ?

**Comment by iamareebjamal** [comment]
Username admin
Password admin

**Comment by iamareebjamal** [comment]
> Oh which means, you are trying to register students in bulk mode? And only two fields in that CSV file? how about rest of the fields like Student nam, Student ID, ..etc

Student registration module is done. It contains all the details of the students. This one maps the courses to the students, i.e., registers students in a particular course

**Comment by jagadeeshkoka** [comment]
So that CSV file should contain Student ID isn’t it ?

**Comment by iamareebjamal** [comment]
Faculty Number

**Comment by jagadeeshkoka** [comment]
Do we have tables created already ?? And where can we have this CSV ?

**Comment by iamareebjamal** [comment]
Yes the tables are already created. CSV has to be made by you. Check the admin panel. Play around with it. You'll understand

**Comment by jagadeeshkoka** [comment]
Hi, Sorry for the late reply. I see you guys have created, Float Course registration and Students registration via CSV file upload, correct me if I am wrong, now with this enhancement we need to map those created courses and Students? if that so what exactly the connecting column for both?

**Comment by iamareebjamal** [comment]
Check the floated course page from department panel section.
